store/models.py

```diff
@@ -17,11 +17,19 @@ def __str__(self):
         return f'{self.title} by {self.author}'
 
 
+class UserRating(models.Model):
+    user=models.ForeignKey(User, related_name='user', null=True, blank=True, on_delete=models.SET_NULL)
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    rating = models.FloatField(default=0)
```

Comment:
The rating shall be given as an integer - please read proper instructions.
The user should not be null here, and a better option would be to use `on_delete=models.CASCADE`

You could have also used `unique_together` META option here.